cc-check.go

```diff
@@ -69,7 +69,11 @@ var requiredKernelModules = map[string]kernelModule{
 	"kvm_intel": {
 		desc: "Intel KVM",
 		parameters: map[string]string{
-			"nested":             "Y",
+			"nested": "Y",
+			// "VMX Unrestricted mode support". This is used
```

Comment:
Reading the comments above, it's unclear to me how the comment should be reworded given the Azure angle. Any suggestions folk?